daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md

```diff
@@ -21,8 +31,8 @@ spec:
     # Connection string
     - name: connectionString
       value: "<CONNECTION STRING>"
-    # Timeout for database operations, in seconds (optional)
-    #- name: timeoutInSeconds
+    # Timeout for database operations, as a Go duration or number of seconds (optional)
+    #- name: timeout
```

Comment:
Presume this is not a breaking change?

Reply:
Correct, the old value is aliased. The new one is more flexible as it allows seconds or a Go duration.